src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Restore/RestoreAzureRMRecoveryServicesBackupItem.cs

```diff
@@ -285,6 +287,18 @@ public class RestoreAzureRmRecoveryServicesBackupItem : RSBackupVaultCmdletBase
         [Parameter(Mandatory = false, ParameterSetName = AzureVMRestoreUnmanagedAsManaged,
             HelpMessage = ParamHelpMsgs.RecoveryPoint.RehydrateDuration)]
         public string RehydrateDuration = "15";
+                
+        [Parameter(Mandatory = false, ParameterSetName = AzureVMManagedDiskParameterSet,
+            HelpMessage = ParamHelpMsgs.RestoreVM.UseMSIForRestore)]
+        public SwitchParameter UseMSIForRestore { get; set; }
```

Comment:
As we discussed in the design review, there's no need for `UseMSIForRestore`, as it can be inferred by the existence of `UseSystemAssignedIdentity ` or `UserAssignedIdentityId `

Reply:
I'll update the PR appropriately.

Reply:
Done